ada/language/ast.py

```diff
@@ -6066,7 +6061,10 @@ def direct_parent_aggregate():
     @langkit_property()
     def xref_equation():
         return If(
-            Self.parent.is_a(AspectClause, AspectAssoc),
+            Or(
```

Comment:
why not `Self.parent.is_a(AspectClause, AspectAssoc, PragmaArgumentAssoc)`?

Reply:
You're right, fixed!